src/backend/tests/endpoints/test_disambiguate.py

```diff
@@ -59,14 +59,10 @@
         "questions": {
             "CASEJD1-1": {
                 "ambiguous_charge_id": "CASEJD1-1",
+                "case_number": "CASEJD1",
                 "answer": "",
                 "options": {"No": "CASEJD1-1-1", "Yes": "CASEJD1-1-0"},
-                "question": "Was the charge "
-                "dismissed "
-                "pursuant to a "
-                "court-ordered "
-                "diversion "
-                "program?",
+                "question": "Was the charge " "dismissed " "pursuant to a " "court-ordered " "diversion " "program?",
```

Comment:
clean up?

Reply:
Black randomly did this. I'm just going to let it be.